README.md

```diff
@@ -1356,7 +1356,7 @@ Other Style Guides
     });
     ```
 
-  - [18.3](#18.3) <a name='18.3'></a> Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). Place no space before the argument list in function calls and declarations.
+  - [18.3](#18.3) <a name='18.3'></a> Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). As well as a space before the argument list in function calls and declarations.
```

Comment:
Just to be clear, function with name stay :
function test() , and not function test ()